apps/site-app/src/containers/App/index.tsx

```diff
@@ -37,30 +39,27 @@ export const App = () => {
   return (
     <MediaContextProvider>
       <style>{mediaStyles}</style>
-      <div id={styles.headerDecoration}>
+
+      <div id={styles.app}>
         <MainNavigation />
-      </div>
 
-      <div className={`${styles.contentContainer}`}>
-        <Container style={{ width: '100%' }} className={``}>
-          <MainPageTransition>
-            <GreaterThanMobileSegment>
-              <Rail internal position='left'>
-                <Sticky context={contextRef}>
-                  <PageSidebar />
-                </Sticky>
-              </Rail>
-            </GreaterThanMobileSegment>
+        <Sidebar />
 
+        <div className={styles.contentContainer}>
+          <MainPageTransition>
             <Ref innerRef={contextRef}>
               <section id={styles.mainContent} className={styles.pageMainInner}>
                 <Routes />
               </section>
             </Ref>
           </MainPageTransition>
-        </Container>
+        </div>
+
+        <BalanceWidget />
+        <ClimateImpactWidget />
```

Comment:
Temporarily set these widgets to constants.  They need to be direct descendents of the grid to work.

When creating new widgets, the pages will need to pass their requested widget back to this component somehow (probably thru prop drilling)

Reply:
The Store / Context combo could be a solution

Reply:
It certainly sounds nicer to use :-)